src/components/utils/test/thread_test.cc

```diff
@@ -54,11 +54,15 @@ const std::string kThreadName("THREAD");
 
 class ThreadTestDelegate : public threads::ThreadDelegate {
  public:
-  ThreadTestDelegate() : thread_stopped_(false), thread_started_(false) {}
+  ThreadTestDelegate()
+      : thread_stopped_(false), thread_started_(false), starts_counter_(0) {}
```

Comment:
@wolfylambova22 I think thread_started_ must be changed to thread_main_executed_. There is a difference between checking that threadFunc() is executed and threadMain() is executed. In both cases thread is started.